test/Microsoft.Azure.WebJobs.EventHubs.UnitTests/PublicSurfaceTests.cs

```diff
@@ -9,7 +9,7 @@ namespace Microsoft.Azure.WebJobs.Host.UnitTests
     public class PublicSurfaceTests
     {
         [Fact]
-        public void EventHubsPublicSurface_LimitedToSpecificTypes()
+        public void WebJobs_Extensions_EventHubs_VerifyPublicSurfaceArea()
```

Comment:
As part of adding missing surface area tests and cleaning them up, I established a naming pattern for these to make viewing them in the test list easier